iOS/DuckDuckGo/TabViewController.swift

```diff
@@ -2488,19 +2499,20 @@ extension TabViewController {
      */
     private func setupOrClearTemporaryDownload(for response: URLResponse) -> WKNavigationResponsePolicy? {
         let downloadManager = AppDependencyProvider.shared.downloadManager
+        let cookieStore = webView.configuration.websiteDataStore.httpCookieStore
         guard response.url != nil,
-              let downloadMetaData = downloadManager.downloadMetaData(for: response),
-              !downloadMetaData.mimeType.isHTML
+              let downloadMetaData = try? downloadManager.downloadMetaData(for: response),
+              !downloadMetaData.mimeType.isHTML,
+              let download = try? downloadManager.makeDownload(response: response,
```

Comment:
Internal checks for preview setup. If they fail, the guard fails, and the existing else block cleans up. The file still renders; no user-facing error needed.